elements/pf-date-picker/pf-calendar.ts

```diff
@@ -0,0 +1,269 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { ref, createRef } from 'lit/directives/ref.js';
+import styles from './pf-calendar.css';
+import { getFormattedDate, getMonthNamesFromLocale } from './date-picker-helper.js';
+import { ComposedEvent } from '@patternfly/pfe-core/core.js';
```

Comment:
prefer to extend `Event` with `bubbles: true` and optionally `cancellable: true`, rather than using `ComposedEvent`

Reply:
Changed `ComposedEvent` to extend `Event` with `bubbles: true `and optionally `cancellable: true`.